src/style.css

```
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url(./images/star.svg) center no-repeat;
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 :first-child,
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(-n+5) {
+  background: url(./images/star-active.svg) center no-repeat;
```

```suggestion
  background: url(./images/star-active.svg);
```
do not repeat yourself